rest-api/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java

```diff
@@ -53,7 +53,7 @@
  * {@link uk.gov.gchq.gaffer.graph.Graph}.
  */
 @Path("/graph/doOperation")
-@Api(value = "/graph/doOperation", description = "Allows operations to be executed on the graph. See <a href='https://github.com/gchq/Gaffer/wiki/operation-examples' target='_blank'>Wiki</a>.")
+@Api(value = "operations", description = "Allows operations to be executed on the graph. See <a href='https://github.com/gchq/Gaffer/wiki/operation-examples' target='_blank'>Wiki</a>.")
```

Comment:
As this name has changed you will need to make the following change to the gaffer.js file:
```
in the addExampleButtons function, the id needs to be changed from 'resource_graphdoOperation' to 'resource_operations'
```